internal/subscription/delay.go

```diff
@@ -0,0 +1,121 @@
+package subscription
```

Comment:
most of the constants belong to retry module we should move them there, for validations we should use validation.go file

Reply:
Will cause cyclic dependency.